.github/workflows/docker.yml

```
@@ -17,14 +17,21 @@ jobs:
-        build: ${{ github.ref == 'refs/heads/main' && github.event_name == 'push' && fromJSON('[{"platform":"linux/amd64","suffix":"-amd64","arch":"amd64"},{"platform":"linux/arm64","suffix":"-arm64","arch":"arm64"}]') || fromJSON('[{"platform":"linux/amd64","suffix":"-amd64","arch":"amd64"}]') }}
-    runs-on: ubuntu-latest
-    timeout-minutes: 300
+    matrix:
+      build:
+        - platform: linux/amd64
+          suffix: -amd64
+          arch: amd64
+        - platform: linux/arm64
+          suffix: -arm64
+          arch: arm64
+    runs-on: ${{ matrix.build.arch == 'arm64' && 'macos-latest' || 'ubuntu-latest' }}
```

actionlint (1.7.8)

property "build" is not defined in object type {}

(expression)